src/renderer/components/database-diagram.jsx

```diff
@@ -84,25 +83,26 @@ export default class DatabaseDiagram extends Component {
 
     try {
       tables.forEach((table, index) => {
-        tableShapes.push(
-          new joint.shapes.sqlectron.Table({
-            position: {
-              x: 100 + (index % 6) * 100,
-              y: 20 + (index % 4) * 100,
-            },
-            size: {
-              width: 120,
-              height: (columnsByTable[table].length + 1.5) * 20,
-            },
-            name: table,
-          }),
-        );
+        const tableShape = new SqlectronShapes.Table({
+          position: {
+            x: 100 + (index % 6) * 100,
+            y: 20 + (index % 4) * 100,
+          },
+          size: {
+            width: 120,
+            height: (columnsByTable[table].length + 1.5) * 20,
+          },
+          name: table,
+        });
+
+        tableShapes.push(tableShape);
```

Comment:
Same code, just moved the table definition into a variable before pushing it into the array